ArduCopter/land_detector.cpp

```diff
@@ -76,7 +76,15 @@ void Copter::update_land_detector()
         // if we have a healthy rangefinder only allow landing detection below 2 meters
         bool rangefinder_check = (!rangefinder_alt_ok() || rangefinder_state.alt_cm_filt.get() < LAND_RANGEFINDER_MIN_ALT_CM);
 
-        if (motor_at_lower_limit && accel_stationary && descent_rate_low && rangefinder_check) {
+#if LANDING_GEAR_ENABLED == ENABLED
+        // if we have weight on wheels (WoW) or ambiguis unknown. never no WoW
+        bool WoW_check = (landinggear.get_wow_state() == 1 || landinggear.get_wow_state() == -1) ? true : false;
+#else
+        // no landing gear so cannot hold up the check
```

Comment:
comment is wrong
no wow sensor so cannot hold up the check

Reply:
Removed